cmd/list.go

```diff
@@ -163,3 +165,25 @@ func printListContainerResponse(lsResponse *common.ListContainerResponse) {
 		glcm.Info(lsResponse.Blobs[index])
 	}
 }
+
+func ByteSizeToString(size int) string {
+	units := []string{
+		"bytes",
+		"kilobytes",
+		"megabytes",
+		"gigabytes",
+		"terabytes",
+		"petabytes",
+		"exabytes", //Let's face it, a file probably won't be more than 1000 exabytes in YEARS.
+	}
+	unit := 0
+	floatSize := float64(size)
+	gigSize := 1024
+
+	for floatSize/float64(gigSize) >= 1 {
+		unit++
+		floatSize /= float64(gigSize)
+	}
+
+	return strconv.FormatFloat(floatSize, 'f', 2, 64) + " " + units[unit]
```

Comment:
please consider adding some unit tests for validation

Reply:
Unit tests added in [9cc9d04](https://github.com/Azure/azure-storage-azcopy/pull/341/commits/9cc9d04ee8a8bc98aee541a17413c42e0f359cfe)